distro/depends/phoronix-test-suite

```diff
@@ -1,4 +1,3 @@
-linux-base
```

Comment:
thanks, why this package is removed?

Reply:
I can't find this package in Fedora repo and can't find this package's information. I guess this may be a legacy problem.

Reply:
does it exist on debian? probably instead of removing it, you can map it as null (empty) on fedora, so we still keep it on debian.

Reply:
You are right! I overlooked debian. Fixed !